src/httpserver.cpp

```diff
@@ -438,6 +430,8 @@ bool InitHTTPServer()
     workQueue = new WorkQueue<HTTPClosure>(workQueueDepth);
     eventBase = base;
     eventHTTP = http;
+    base_ctr.release();
+    http_ctr.release();
```

Comment:
I mean,  I guess it improves consistency, and the `.release()`,  if you added a comment noting the above,  would be clearer than what was there before.